package.go

```diff
@@ -180,7 +180,77 @@ func (c *contentTypes) addDefault(extension, contentType string) {
 	c.defaults[extension] = contentType
 }
 
-type corePropertiesXML struct {
+func (c *contentTypes) findType(name string) (string, error) {
+	if t, ok := c.overrides[strings.ToUpper(name)]; ok {
+		return t, nil
+	}
+	ext := filepath.Ext(name)
+	if ext != "" {
+		if t, ok := c.defaults[ext[1:]]; ok {
```

Comment:
what if extension is upper case and default is lower case? The spec says the comparations should be done case insensitive. Is there a unit test for it?

Reply:
I've corrected and added a test for this case, that also check the override case insensitive.